modape/modis/window.py

```diff
@@ -139,18 +139,23 @@ def generate_mosaics(self,
             labels = None
             force_doy = True
 
-        output_res = [None, None]
         if "xRes" in kwargs and "yRes" in kwargs:
             output_res = [kwargs["xRes"], kwargs["yRes"]]
             del kwargs["xRes"]
             del kwargs["yRes"]
 
         elif target_srs == "EPSG:4326":
 
-            if not attrs["globalproduct"]:
-                output_res = attrs["resolution"] / 112000
-            else:
-                output_res = attrs["resolution"]
+            try:
+                if not attrs["globalproduct"]:
+                    output_res = attrs["resolution"] / 112000
```

Comment:
We default back now to `[None, None]` if none of the conditions apply or if we can't read the info from file.